### ISSUE TITLE
Broken link from the html form project 

### ISSUE THREAD

**Issue by Sahmie** [TheOdinProject/curriculum#16063]
came across a dead link from the Project: HTML forms in the web dev 101 section 
https://www.theodinproject.com/courses/html5-and-css3/lessons/html-forms

The link to the mint.com website doesn't connect on my end but other links go through

------

We should find another link to add to this section for people who cannot view the mint.com link. 

Steps:
- [ ] Find link of equal value
- [ ] Test link against multiple countries (specifically ones that were failing to load the current link.
- [ ] Add the link to the lesson alongside the mint.com link.

**Comment by I3uckwheat** [comment]
I wonder if this has to do with region locking or something. This link works for me, but can a few people outside the US take a peek?

**Comment by AJMcDee** [comment]
@I3uckwheat It works for me here in Germany

**Comment by I3uckwheat** [comment]
@AJMcDee Thanks for taking a look!

**Comment by ARodrigues92** [comment]
@I3uckwheat Working fine as well in Portugal

**Comment by NikSeth** [comment]
The link is working for me. Nation: India

**Comment by I3uckwheat** [comment]
@ARodrigues92 @NikSeth 

Thanks guys!

**Comment by Theoinf0** [comment]
All good here in UK

**Comment by Theoinf0** [comment]
All good here in UK

**Comment by aaron-contreras** [comment]
Panama working A-OK!

**Comment by zephyrasy** [comment]
The link to the mint.com doesn't work for me in Iran.

**Comment by NikSeth** [comment]
Can the link be changed to a link containing the screenshot of the sign-up form?

**Comment by I3uckwheat** [comment]
@NikSeth Part of forums is understanding how they function when interacting with them.

@zephyrasy Thanks, sounds like there are some regions that this does not work with. We'll look for a replacement.

**Comment by mKawi** [comment]
I'm happy to take this. Quick question, should the new link replace the signup page for Mint.com? 
Or rather adding it as a secondary backup link?

**Comment by muhymenulhaque** [comment]
mint.com works but https://wwws.mint.com/login.event?task=S redirects me to https://accounts.intuit.com/signup.html ... 
Is it normal?

**Comment by xandora** [comment]
@muhymenulhaque Yes, Intuit Inc. owns Mint. They have a centralised logon page for their various services.

**Comment by Timework** [comment]
Just a question, can wayback machine get passed the region limitation. 
If so we can just use this
https://web.archive.org/web/20200330102121/https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb
and tell them to click the little x for the wayback machine thing on the top

**Comment by I3uckwheat** [comment]
@Timework I don't think we want to use that for something like this. We should be current as possible.

**Comment by I3uckwheat** [comment]
> I'm happy to take this. Quick question, should the new link replace the signup page for Mint.com?
> Or rather adding it as a secondary backup link?

I would add it as a secondary backup link. I'll assign this issue to you! Thanks for helping!

**Comment by mKawi** [comment]
Made a [PR](https://github.com/TheOdinProject/curriculum/pull/16314).

I chose Mailchimp's signup form as it's not overly complicated, yet still challenging enough to learn from. The focus is still on the form rather than on other UI.

I used [UpTrends](https://www.uptrends.com/tools/uptime) and [App Synthetic Monitor](https://asm.ca.com/en/ping.php) to check if the site is available across the globe. It's available in Iran, which a user here said they had trouble accessing Mint.com from.

Let me know what changes to make, hope I did well for my first pull request :)
